pkg/solver/matcher.go

```diff
@@ -143,6 +144,32 @@ func getMatchingDeals(
 	// loop over job offers
 	for _, jobOffer := range jobOffers {
 
+		// See if our jobOffer targets a specific address. If so, we will create a deal automatically
+		// with the matcing resourceOffer.
+		if jobOffer.JobOffer.Target.Address != "" {
+			resourceOffer, err := db.GetResourceOfferByAddress(jobOffer.JobOffer.Target.Address)
+			if err != nil {
+				return nil, err
```

Comment:
What do you think about adding some logging for this error block and the one on line 167

Reply:
This is a little future guarding - right now `GetResourceOfferByAddress` will never return an error (because we only have an in-memory store so far). so it either finds it or it doesn't, but won't throw an error.